internal/commands/user/revoke.go

```diff
@@ -0,0 +1,119 @@
+package user
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/10gen/realm-cli/internal/utils/flags"
+
+	"github.com/spf13/pflag"
+)
+
+// CommandRevoke is the `user revoke` command
+type CommandRevoke struct {
+	inputs      revokeInputs
+	outputs     userOutputs
+	realmClient realm.Client
+}
+
+type revokeInputs struct {
```

Comment:
[nit] should this just follow the same pattern as the command and be `inputsRevoke`?

Reply:
we discussed this one a while back together... we kept inputs in the style of `xxxInputs` to match the corresponding file `xxx_inputs.go` (ensuring inputs and command code sat next to each other in the package)